src/containers/SearchHit.js

```diff
@@ -45,13 +45,11 @@ const mapDispatchToProps = {
 /** */
 const styles = theme => ({
   adjacent: {},
-  canvasLabel: {
-    ...theme.typography.h6,
-  },
   focused: {},
   hitCounter: {
-    ...theme.typography.h6,
+    ...theme.typography.subtitle2,
     backgroundColor: theme.palette.hitCounter.default,
+    height: '30',
```

Comment:
I'm seeing an issue with this not getting applied... maybe it needs to be an Integer or `'30px'`?

<img width="127" alt="Screen Shot 2019-06-18 at 5 07 29 PM" src="https://user-images.githubusercontent.com/1656824/59725652-adc19d80-91eb-11e9-8a02-883fa54caff8.png">